ingestion/src/main.rs

```diff
@@ -39,6 +39,13 @@ async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
     ));
     let background_permit_tables = permit_tables.clone();
 
+    // Levels tables handling (needs connection to stinfosys database)
+    let level_table = Arc::new(RwLock::new(
+        levels::fetch_levels(&stinfo_conn_string).await?,
+    ));
+    // maybe need this later?
+    let _background_level_table = level_table.clone();
```

Comment:
I just noticed we aren't refreshing this in the background like we are the permit tables. We should probably do that.

Reply:
Nominally added in last commit, but not with generics and such as mentioned.

Reply:
We can take that up another time! Thanks for leaving a TODO